src/parser/core/modules/ResourceGraphs/Graph.tsx

```diff
@@ -44,8 +44,8 @@ export function Graph({resource, scaleX}: GraphProps) {
 				data={data}
 				curve={(resource.linear ?? false) ? curveLinear : curveStepAfter}
 				x={datum => scaleX(datum.time)}
-				y0={scaleY(0)}
-				y1={datum => scaleY(datum.current)}
+				y0={datum => scaleY(datum.base || 0)}
+				y1={datum => scaleY((datum.base || 0) + datum.current)}
```

Comment:
given these are `T | undefined`, can write as `datum.base ?? 0`

i recognise that the only case that `||` would trip on here is `0` which is also the fall-through, but _semantics_

Reply:
Always have trouble remembering which to do and why. Fixed.